packages/forms/resources/views/components/section.blade.php

```diff
@@ -25,22 +25,36 @@
         ])
         @if ($isCollapsible())
             x-bind:class="{ 'rounded-b-xl': isCollapsed }"
+            x-on:click="
+                isCollapsed = ! isCollapsed
+                if (! isCollapsed) {
+                    setTimeout(
```

Comment:
What's the `setTimeout` for?

Reply:
Only cosmetics. Not needed to be functional.

Reply:
Hmm, could you post a video of the difference?

Reply:
Tomorrow :)

Reply:
I think the `setTimeout` is a good addition in this case, thank you